app/controllers/project_controller.rb

```diff
@@ -294,17 +294,29 @@ def admin_request
   # "Posts" to the same action.  Stays on this view until done.
   def add_members
     pass_query_params
-    if @project = find_or_goto_index(Project, params[:id].to_s)
-      @users = User.where.not(verified: nil).order("login, name").to_a
-      if !@project.is_admin?(@user)
-        redirect_with_query(action: "show_project", id: @project.id)
-      elsif params[:candidate].present?
-        @candidate = User.find(params[:candidate])
-        set_status(@project, :member, @candidate, :add)
-      end
+    return unless @project = find_or_goto_index(Project, params[:id].to_s)
```

Comment:
Sorry I didn't notice this earlier
```suggestion
    return unless (@project = find_or_goto_index(Project, params[:id].to_s))
```